docs/installation/community-operator.md

```diff
@@ -1,11 +1,21 @@
 # Kepler Community Operator on OpenShift
 
-## Install operator from Operator Hub
+## Requirements
 
-Before you start make sure you have a OCP 4.13 cluster running.
+Before you start make sure you have:
 
+- An OCP 4.13 cluster running
+- Signed in as `kubeadmin` or a user with `cluster-admin` role
+- `oc` installed.
+- Cloned the [kepler-operator](https://github.com/sustainable-computing-io/kepler-operator/) repository.
+```sh
+git clone https://github.com/sustainable-computing-io/kepler-operator.git
+cd kepler-exporter
+```
+---
```

Comment:
Now that I think about it, we should provide an alternative way that doesn't require you to have 'oc' installed.  It takes more steps to do in, but it can all be done in OCP Web console.  This request be done in a separate PR.